e2e_test/e2e_test.go

```diff
@@ -309,7 +309,6 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 			nfIngressIp                string
 			nfEgressIp                 string
 			externalClientIp           string
-			externalClientDev          string
```

Comment:
Why does the same PR re-introduce a previously removed test, and then deletes it again.

Probably, don't do that.

If you really want to do this, then please explain in the commit message why this is done.